.grabl/automation.yml

```diff
@@ -79,7 +79,6 @@ build:
       dependencies: [build, build-dependency, test-behaviour]
     test-deployment-maven:
       machine: graknlabs-ubuntu-20.04
-      dependencies: [deploy-maven-snapshot]
```

Comment:
Why are we removing this dependency, @alexjpwalker ? Don't we need to make sure that `deploy-maven-snapshot` is successful before we test the maven deployment? Would be good to add comments to this PR btw.